internal/db/schema/migrations/postgres/14/01_wh_user_dimension_oidc.up.sql

```diff
@@ -0,0 +1,154 @@
+begin;
+
+  alter table wh_user_dimension
+    add column auth_method_external_id  wh_dim_text,
+    add column auth_account_external_id wh_dim_text,
+    add column auth_account_full_name   wh_dim_text,
+    add column auth_account_email       wh_dim_text
+  ;
+
+  drop view whx_user_dimension_source;
+  create view whx_user_dimension_source as
+       select -- id is the first column in the target view
+              u.public_id                       as user_id,
+              coalesce(u.name, 'None')          as user_name,
+              coalesce(u.description, 'None')   as user_description,
+              coalesce(aa.public_id, 'None')    as auth_account_id,
+              case
+                   when apa.public_id is not null then 'password auth account'
+                   when aoa.public_id is not null then 'oidc auth account'
+                   else 'None'
+                   end                          as auth_account_type,
+              coalesce(apa.name, 'None')        as auth_account_name,
+              coalesce(apa.description, 'None') as auth_account_description,
+              case
+                  when apa.public_id is not null then 'Not Applicable'
+                  when aoa.public_id is null then 'None'
+                  else aoa.subject
+                  end                           as auth_account_external_id,
+              case
+                  when apa.public_id is not null then 'Not Applicable'
+                  when  aoa.public_id is not null
+                    and aoa.full_name is not null then aoa.full_name
+                  else 'None'
+                  end                           as auth_account_full_name,
+              case
+                  when apa.public_id is not null then 'Not Applicable'
+                  when  aoa.public_id is not null
+                    and aoa.email is not null then aoa.email
+                  else 'None'
+                  end                           as auth_account_email,
+              coalesce(am.public_id, 'None')    as auth_method_id,
+              case
+                   when apa.public_id is not null then 'password auth method'
+                   when aoa.public_id is not null then 'oidc auth method'
+                   else 'None'
+                   end                          as auth_method_type,
+              coalesce(apm.name, 'None')        as auth_method_name,
+              coalesce(apm.description, 'None') as auth_method_description,
```

Comment:
The `auth_oidc_method` table has `name` and `description` columns so these should be `case` statements too.